### PR TITLE
Create queries for Device Boot Attestation Info blog

### PR DESCRIPTION
**Pull request by FlyingBlueMonkey**
Device Boot Attestation Info queries to show System Guard Security Level

### CODE REVIEW

File: General queries/System Guard Security Level Baseline.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Establishes a baseline SystemGuardSecurityLevel and show the devices that are below that baseline
```
Comment (meni-braun) [inline General queries/System Guard Security Level Baseline.txt]: Any chance you add a comment on what this baseline mean? Maybe point to a doc URL?

File: General queries/System Guard Security Level Baseline.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+// For more questions on this query, feel free to ping @DepletionMode or @flyingbluemonki on twitter
+let TargetSecurityLevel = 700;
+MiscEvents
+| where EventTime >= ago(30d)
```
Comment (meni-braun) [inline General queries/System Guard Security Level Baseline.txt]: I prefer to filter all queries by 7d by default.
Mind to change it?

File: General queries/System Guard Security Level Drop.txt [hunk 3]
```diff
@@ -0,0 +1,24 @@
+// System Guard Security Level Drop
+// This query looks at the value of SystemGuardSecurityLevel in the DeviceBootAttestationInfo action type records in MiscEvents
+// when it sees a drop in the value of SystemGuardSecurityLevel of the latest record verus the previous record
+// You can convert this into a custom detection by selecting the "Create Detection Rule" in Advanced Hunting
+// For more questions on this query, feel free to ping @DepletionMode or @FlyingBlueMonki on twitter
+MiscEvents
+| where ActionType == "DeviceBootAttestationInfo"
+| partition by MachineId
```
Comment (meni-braun) [inline General queries/System Guard Security Level Drop.txt]: I got an error when trying to run it on a large tenant:
`'partition' operator: exceeded amount of maximum partitions allowed (64)`

File: Protection events/ExploitGuardNetworkProtectionEvents.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+//Simple query to show the unique network connections that were audited or blocked by ExploitGuard
+// For more questions on this query, feel free to ping @FlyingBlueMonki on twitter or mattegen@microsoft.com via email
+MiscEvents
+| where EventTime > ago(7d)
+| where  ActionType  =~ "ExploitGuardNetworkProtectionBlocked"
+| summarize count(RemoteUrl) by InitiatingProcessFileName, RemoteUrl, Audit_Only=tostring(parse_json(AdditionalFields).IsAudit)
```
Comment (meni-braun) [inline Protection events/ExploitGuardNetworkProtectionEvents.txt]: When `IsAudit`=true, doesn't it mean 'blocked'?
If so, I suggest you change it to `IsBlocked`

Comment (FlyingBlueMonkey) [reply Protection events/ExploitGuardNetworkProtectionEvents.txt]: Not sure what you mean here.  When "IsAudit" = true it means it was not blocked but only audited.

File: General queries/System Guard Security Level Drop.txt (hunk 3 above)
Comment (FlyingBlueMonkey) [reply General queries/System Guard Security Level Drop.txt]: I will look into this one and see what's going on.

**Pull request merged**